src/main/java/assets/css/custom.css

```diff
@@ -1,6 +1,14 @@
 /* Add additional stylesheets below
 -------------------------------------------------- */
 
+.banner{
+    text-align: center;
+}
+
+.background{
+    background-color: #FF0266;
```

Comment:
this color is gross, it makes reading the black text on blue background far too hard to read